README.md

```diff
@@ -1,27 +1,4 @@
 <a name="readme-top"></a>
-
-<!--
-HOW TO USE:
-This is an example of how you may give instructions on setting up your project locally.
-
-Modify this file to match your project and remove sections that don't apply.
-
-REQUIRED SECTIONS:
-- Table of Contents
-- About the Project
-  - Built With
-  - Live Demo
-- Getting Started
```

Comment:
- [ ] Kindly make sure your project has a descriptive summary telling in detail what changes were made.
![image](https://user-images.githubusercontent.com/97100091/204557767-c94e6e06-2cb5-40e8-a749-e5d3a4897cc5.png)

You may consider clicking those three dots on the right and from there you will be able to edit. :smile: